### PR TITLE
Bug 1879445: Add a route annotation for Samesite

### PR DESCRIPTION
**Pull request by miheer**
1. Add a route annotation for Samesite
https://issues.redhat.com/browse/RFE-600
Bug 1879445 - SameSite hardening breaks sticky load balancing

2. Adding notes that the upstream PR breaks product builds and we now have an "official" way to get the latest haproxy RPM
we reverted their PR because it breaks Red Hat product builds

@jupierce @frobware

### CODE REVIEW

**Comment by frobware** [comment]
As it has been 15 days since the last CI run:

/test unit

**Comment by frobware** [comment]
Testing again in CI but also adding a _hold_ as we have necessary changes in this PR to the Dockerfile that should not be commited. Those changes involve pulling in a temporary build of haproxy-1.8.26 which this change depends on.

/hold

**Comment by frobware** [comment]
Differences between failures in this CI which uses the new haproxy-1.8.26 RPM and a no-change PR to assert that this PR does not introduce new or different failures are (zero) :

```diff
2020/10/20 09:16:34 ci-operator version v20201015-b9bdbe6     |	2020/10/05 03:27:25 ci-operator version v20201002-9e95fb1
2020/10/20 09:16:34 Loading configuration from https://config |	2020/10/05 03:27:25 Loading configuration from https://config
2020/10/20 09:16:34 Resolved source https://github.com/opensh |	2020/10/05 03:27:25 Resolved source https://github.com/opensh
2020/10/20 09:16:34 Resolved openshift/release:golang-1.10 to |	2020/10/05 03:27:25 Resolved openshift/release:golang-1.10 to
2020/10/20 09:16:34 Resolved openshift/centos:7 to sha256:fe2 |	2020/10/05 03:27:25 Resolved openshift/centos:7 to sha256:fe2
2020/10/20 09:16:34 Using namespace https://console.build01.c |	2020/10/05 03:27:25 Using namespace https://console-openshift
2020/10/20 09:16:34 Running [input:root], src, unit	      |	2020/10/05 03:27:25 Running [input:root], src, unit
2020/10/20 09:16:35 Creating namespace ci-op-nl6pmbkq	      |	2020/10/05 03:27:25 Creating namespace ci-op-rh6hzqn9
2020/10/20 09:16:35 Creating rolebinding for user miheer in n |	2020/10/05 03:27:25 Creating rolebinding for user miheer in n
2020/10/20 09:16:35 Setting up pipeline imagestream for the t |	2020/10/05 03:27:25 Setting up pipeline imagestream for the t
2020/10/20 09:16:35 Created PDB for pods with openshift.io/bu |	2020/10/05 03:27:25 Created PDB for pods with openshift.io/bu
2020/10/20 09:16:35 Created PDB for pods with created-by-ci l |	2020/10/05 03:27:25 Created PDB for pods with created-by-ci l
2020/10/20 09:16:35 Tagging openshift/release:golang-1.10 int |	2020/10/05 03:27:25 Tagging openshift/release:golang-1.10 int
2020/10/20 09:16:35 Building src			      |	2020/10/05 03:27:25 Building src
2020/10/20 09:31:36 Build src succeeded after 16m38s	      |	2020/10/05 03:32:15 Build src succeeded after 4m50s
2020/10/20 09:31:36 Executing test unit			      |	2020/10/05 03:32:15 Executing test unit
2020/10/20 09:31:36 Executing pod "unit"		      <
[INFO] [CLEANUP] Cleaning up temporary directories		[INFO] [CLEANUP] Cleaning up temporary directories
[INFO] Running `go test`...					[INFO] Running `go test`...
[INFO] No compiled `gotest2junit` binary was found. Attemptin	[INFO] No compiled `gotest2junit` binary was found. Attemptin
[INFO]   $ hack/build-go.sh tools/gotest2junit			[INFO]   $ hack/build-go.sh tools/gotest2junit
++ Building go targets for linux/amd64: tools/gotest2junit	++ Building go targets for linux/amd64: tools/gotest2junit
[INFO] hack/build-go.sh exited with code 0 after 00h 00m 05s	[INFO] hack/build-go.sh exited with code 0 after 00h 00m 05s
FAIL: github.com/openshift/origin/pkg/build/builder TestDocke	FAIL: github.com/openshift/origin/pkg/build/builder TestDocke
FAIL: github.com/openshift/origin/pkg/build/builder TestDocke	FAIL: github.com/openshift/origin/pkg/build/builder TestDocke
FAIL: github.com/openshift/origin/pkg/build/builder TestDocke	FAIL: github.com/openshift/origin/pkg/build/builder TestDocke
FAIL: github.com/openshift/origin/pkg/build/builder TestPushE	FAIL: github.com/openshift/origin/pkg/build/builder TestPushE
FAIL: github.com/openshift/origin/pkg/build/builder TestGetSt	FAIL: github.com/openshift/origin/pkg/build/builder TestGetSt
SKIP: github.com/openshift/origin/pkg/oc/cli/admin/prune/imag	SKIP: github.com/openshift/origin/pkg/oc/cli/admin/prune/imag
SKIP: github.com/openshift/origin/pkg/templateservicebroker/s	SKIP: github.com/openshift/origin/pkg/templateservicebroker/s
SKIP: github.com/openshift/origin/vendor/k8s.io/kubernetes/pk	SKIP: github.com/openshift/origin/vendor/k8s.io/kubernetes/pk
SKIP: github.com/openshift/origin/vendor/k8s.io/kubernetes/pk	SKIP: github.com/openshift/origin/vendor/k8s.io/kubernetes/pk
SKIP: github.com/openshift/origin/vendor/k8s.io/kubernetes/pk	SKIP: github.com/openshift/origin/vendor/k8s.io/kubernetes/pk
SKIP: github.com/openshift/origin/vendor/k8s.io/kubernetes/pk	SKIP: github.com/openshift/origin/vendor/k8s.io/kubernetes/pk
SKIP: github.com/openshift/origin/vendor/k8s.io/kubernetes/pk	SKIP: github.com/openshift/origin/vendor/k8s.io/kubernetes/pk
SKIP: github.com/openshift/origin/vendor/k8s.io/kubernetes/pk	SKIP: github.com/openshift/origin/vendor/k8s.io/kubernetes/pk
SKIP: github.com/openshift/origin/vendor/k8s.io/kubernetes/pk	SKIP: github.com/openshift/origin/vendor/k8s.io/kubernetes/pk
SKIP: github.com/openshift/origin/vendor/k8s.io/kubernetes/pk	SKIP: github.com/openshift/origin/vendor/k8s.io/kubernetes/pk
SKIP: github.com/openshift/origin/vendor/k8s.io/kubernetes/pk	SKIP: github.com/openshift/origin/vendor/k8s.io/kubernetes/pk
SKIP: github.com/openshift/origin/vendor/k8s.io/kubernetes/pk	SKIP: github.com/openshift/origin/vendor/k8s.io/kubernetes/pk
SKIP: github.com/openshift/origin/vendor/k8s.io/kubernetes/pk	SKIP: github.com/openshift/origin/vendor/k8s.io/kubernetes/pk
SKIP: github.com/openshift/origin/vendor/k8s.io/kubernetes/pk	SKIP: github.com/openshift/origin/vendor/k8s.io/kubernetes/pk
SKIP: github.com/openshift/origin/vendor/k8s.io/kubernetes/pk	SKIP: github.com/openshift/origin/vendor/k8s.io/kubernetes/pk
SKIP: github.com/openshift/origin/vendor/k8s.io/kubernetes/pk	SKIP: github.com/openshift/origin/vendor/k8s.io/kubernetes/pk
SKIP: github.com/openshift/origin/vendor/k8s.io/kubernetes/pk	SKIP: github.com/openshift/origin/vendor/k8s.io/kubernetes/pk
[ERROR] hack/test-go.sh exited with code 1 after 01h 10m 27s  |	[ERROR] hack/test-go.sh exited with code 1 after 01h 11m 55s
2020/10/20 10:42:57 Container test in pod unit failed, exit c |	2020/10/05 04:45:06 Container test in pod unit failed, exit c
2020/10/20 10:42:57 Copied 2.46MB of artifacts from unit to / |	2020/10/05 04:45:06 Copied 2.44MB of artifacts from unit to /
2020/10/20 10:42:57 No custom metadata found and prow metadat |	2020/10/05 04:45:06 No custom metadata found and prow metadat
2020/10/20 10:42:58 Ran for 1h26m23s			      |	2020/10/05 04:45:06 Ran for 1h17m41s
error: some steps failed:					error: some steps failed:
  * could not run steps: step unit failed: test "unit" failed |	  * could not run steps: step unit failed: test "unit" failed

Container test exited with code 1, reason Error			Container test exited with code 1, reason Error
---								---
uilder TestDockerBuildError 0s					uilder TestDockerBuildError 0s
FAIL: github.com/openshift/origin/pkg/build/builder TestPushE	FAIL: github.com/openshift/origin/pkg/build/builder TestPushE
FAIL: github.com/openshift/origin/pkg/build/builder TestGetSt	FAIL: github.com/openshift/origin/pkg/build/builder TestGetSt
SKIP: github.com/openshift/origin/pkg/oc/cli/admin/prune/imag	SKIP: github.com/openshift/origin/pkg/oc/cli/admin/prune/imag
SKIP: github.com/openshift/origin/pkg/templateservicebroker/s	SKIP: github.com/openshift/origin/pkg/templateservicebroker/s
SKIP: github.com/openshift/origin/vendor/k8s.io/kubernetes/pk	SKIP: github.com/openshift/origin/vendor/k8s.io/kubernetes/pk
SKIP: github.com/openshift/origin/vendor/k8s.io/kubernetes/pk	SKIP: github.com/openshift/origin/vendor/k8s.io/kubernetes/pk
SKIP: github.com/openshift/origin/vendor/k8s.io/kubernetes/pk	SKIP: github.com/openshift/origin/vendor/k8s.io/kubernetes/pk
SKIP: github.com/openshift/origin/vendor/k8s.io/kubernetes/pk	SKIP: github.com/openshift/origin/vendor/k8s.io/kubernetes/pk
SKIP: github.com/openshift/origin/vendor/k8s.io/kubernetes/pk	SKIP: github.com/openshift/origin/vendor/k8s.io/kubernetes/pk
SKIP: github.com/openshift/origin/vendor/k8s.io/kubernetes/pk	SKIP: github.com/openshift/origin/vendor/k8s.io/kubernetes/pk
SKIP: github.com/openshift/origin/vendor/k8s.io/kubernetes/pk	SKIP: github.com/openshift/origin/vendor/k8s.io/kubernetes/pk
SKIP: github.com/openshift/origin/vendor/k8s.io/kubernetes/pk	SKIP: github.com/openshift/origin/vendor/k8s.io/kubernetes/pk
SKIP: github.com/openshift/origin/vendor/k8s.io/kubernetes/pk	SKIP: github.com/openshift/origin/vendor/k8s.io/kubernetes/pk
SKIP: github.com/openshift/origin/vendor/k8s.io/kubernetes/pk	SKIP: github.com/openshift/origin/vendor/k8s.io/kubernetes/pk
SKIP: github.com/openshift/origin/vendor/k8s.io/kubernetes/pk	SKIP: github.com/openshift/origin/vendor/k8s.io/kubernetes/pk
SKIP: github.com/openshift/origin/vendor/k8s.io/kubernetes/pk	SKIP: github.com/openshift/origin/vendor/k8s.io/kubernetes/pk
SKIP: github.com/openshift/origin/vendor/k8s.io/kubernetes/pk	SKIP: github.com/openshift/origin/vendor/k8s.io/kubernetes/pk
SKIP: github.com/openshift/origin/vendor/k8s.io/kubernetes/pk	SKIP: github.com/openshift/origin/vendor/k8s.io/kubernetes/pk
SKIP: github.com/openshift/origin/vendor/k8s.io/kubernetes/pk	SKIP: github.com/openshift/origin/vendor/k8s.io/kubernetes/pk
[ERROR] hack/test-go.sh exited with code 1 after 01h 10m 27s  |	[ERROR] hack/test-go.sh exited with code 1 after 01h 11m 55s
---								---
time="2020-10-20T10:42:58Z" level=info msg="Reporting job sta |	time="2020-10-05T04:45:06Z" level=info msg="Reporting job sta
```

**Comment by frobware** [comment]
This is a backport of https://github.com/openshift/router/pull/189

**Comment by frobware** [comment]
The build failures here look to be similar to https://github.com/openshift/origin/issues/24874#issuecomment-614209663

**Comment by frobware** [comment]
Expecting CI failures until the new haproxy-1.8.26 is available in http://download.eng.bos.redhat.com/rcm-guest/puddles/RHAOS/AtomicOpenShift/3.11/latest/x86_64/os/Packages

**Comment by frobware** [comment]
haproxy-1.8.26 brew build: https://brewweb.engineering.redhat.com/brew/buildinfo?buildID=1360532

**Comment by miheer** [comment]
https://github.com/openshift/origin/pull/25626  @Miciah @frobware  can we have this fix merged ASAP ?  The CI is failing due to wrong file path added.

**Comment by frobware** [comment]
> Testing again in CI but also adding a _hold_ as we have necessary changes in this PR to the Dockerfile that should not be commited. Those changes involve pulling in a temporary build of haproxy-1.8.26 which this change depends on.
> 
> /hold

/hold cancel

haproxy-1.8.26 is now in http://download.eng.bos.redhat.com/rcm-guest/puddles/RHAOS/AtomicOpenShift/3.11/latest/x86_64/os/Packages/?C=M;O=D

**Comment by frobware** [comment]
> #25626 @Miciah @frobware @knobunc can we have this fix merged ASAP ? The CI is failing due to wrong file path added.

Does it fail CI? I ask because https://github.com/openshift/origin/pull/25616 just passed/merged.

**Comment by miheer** [comment]
@frobware The earlier CI unit test is passing and other tests eg- https://prow.ci.openshift.org/view/gcs/origin-ci-test/pr-logs/pull/25542/test_pull_request_origin_extended_conformance_install-release-3.11/2152/  is failing where I can see the router is failing to come up
~~~
}
stderr:
cat: /var/lib//var/lib/haproxy/conf/haproxy.config: No such file or directory
command terminated with exit code 1
error:
exit status 1
Oct 23 05:20:37.520: INFO: Running '/data/src/github.com/openshift/origin/_output/local/bin/linux/amd64/kubectl --server=https://ip-172-18-1-172.ec2.internal:8443 --kubeconfig=/etc/origin/master/admin.kubeconfig exec --namespace=default router-1-jfnzd -- /bin/sh -c cat /var/lib/haproxy/router/routes.json /var/lib//var/lib/haproxy/conf/haproxy.config'
Oct 23 05:20:37.860: INFO: rc: 1
[AfterEach] [Conformance][Area:Networking][Feature:Router]
skipped 88 lines unfold_more
  /go/src/github.com/openshift/origin/_output/local/go/src/github.com/openshift/origin/vendor/k8s.io/kubernetes/test/e2e/framework/framework.go:142
STEP: Dumping a list of prepulled images on each node to file /data/src/github.com/openshift/origin/_output/scripts/conformance/artifacts/junit/image-puller.txt
Oct 23 05:20:57.939: INFO: Waiting up to 3m0s for all (but 0) nodes to be ready
• Failure [510.533 seconds]
[Conformance][Area:Networking][Feature:Router]
/go/src/github.com/openshift/origin/_output/local/go/src/github.com/openshift/origin/test/extended/router/router.go:20
  The HAProxy router
  /go/src/github.com/openshift/origin/_output/local/go/src/github.com/openshift/origin/test/extended/router/router.go:56
    should serve routes that were created from an ingress [Suite:openshift/conformance/parallel/minimal] [It]
    /go/src/github.com/openshift/origin/_output/local/go/src/github.com/openshift/origin/test/extended/router/router.go:67
    Expected error:
        <*errors.errorString | 0xc4200c9590>: {
            s: "timed out waiting for the condition",
        }
        timed out waiting for the condition
    not to have occurred
    /go/src/github.com/openshift/origin/_output/local/go/src/github.com/openshift/origin/test/extended/util/url/url.go:134
------------------------------
Oct 23 05:20:58.059: INFO: Running AfterSuite actions on all node
skipped 26 lines unfold_more
[Fail] [Conformance][Area:Networking][Feature:Router] The HAProxy router [It] should serve routes that were created from an ingress [Suite:openshift/conformance/parallel/minimal] 
/go/src/github.com/openshift/origin/_output/local/go/src/github.com/openshift/origin/test/extended/util/url/url.go:134
Ran 228 of 1270 Specs in 1934.309 seconds
FAIL! -- 223 Passed | 5 Failed | 0 Pending | 1042 Skipped 
Ginkgo ran 1 suite in 32m16.343427735s
Test Suite Failed
[INFO] Running serial tests
I1023 05:20:58.928207   26793 test.go:85] Extended test version v3.11.0+1202ad0-5
~~~

**Comment by frobware** [comment]
The last e2e-gcp failed because the haproxy version isn't 1.8.26 - https://gcsweb-ci.apps.ci.l2s4.p1.openshiftapps.com/gcs/origin-ci-test/pr-logs/pull/25542/pull-ci-openshift-origin-release-3.11-e2e-gcp/1319487697525411840/artifacts/e2e-gcp/pods/default_router-1-cgpbw_router.log

haproxy fails to start as the template change in this PR requires 1.8.26 but that should be available now as we see it listed here http://download.eng.bos.redhat.com/rcm-guest/puddles/RHAOS/AtomicOpenShift/3.11/latest/x86_64/os/Packages/?C=M;O=D but maybe there's some lag (or other) before it becomes available in a CI run.

**Comment by frobware** [comment]
> Does it fail CI? I ask because #25616 just passed/merged.

The failures are not connected but it would be good to have this anyway. 

e2e-gcp is failing because we don't yet have haproxy-1.8.26 for CI runs:  https://github.com/openshift/origin/pull/25542#issuecomment-715256191

**Comment by frobware** [comment]
Looking through https://storage.googleapis.com/origin-ci-test/pr-logs/pull/25542/test_pull_request_origin_end_to_end_311/1620/build-log.txt it seems that we get haproxy18-1.8.1 which ancient,. 

```
3277:[openshift/origin-haproxy-router] --> RUN INSTALL_PKGS="haproxy18 rsyslog" &&     yum install -y $INSTALL_PKGS &&     rpm -V $INSTALL_PKGS &&     yum clean all &&     mkdir -p /var/lib/haproxy/router/{certs,cacerts,whitelists} &&     mkdir -p /var/lib/haproxy/{conf/.tmp,run,bin,log} &&     touch /var/lib/haproxy/conf/{{os_http_be,os_edge_reencrypt_be,os_tcp_be,os_sni_passthrough,os_route_http_redirect,cert_config,os_wildcard_domain}.map,haproxy.config} &&     setcap 'cap_net_bind_service=ep' /usr/sbin/haproxy &&     chown -R :0 /var/lib/haproxy &&     chmod -R g+w /var/lib/haproxy
3285:[openshift/origin-haproxy-router] ---> Package haproxy18.x86_64 0:1.8.1-5.el7 will be installed
3301:[openshift/origin-haproxy-router]  haproxy18   x86_64 1.8.1-5.el7   cbs-paas7-openshift-multiarch-el7-build 1.2 M
3324:[openshift/origin-haproxy-router]   Installing : haproxy18-1.8.1-5.el7.x86_64                                 5/5 
3328:[openshift/origin-haproxy-router]   Verifying  : haproxy18-1.8.1-5.el7.x86_64                                 4/5 
3331:[openshift/origin-haproxy-router]   haproxy18.x86_64 0:1.8.1-5.el7       rsyslog.x86_64 0:8.24.0-52.el7_8.2 
```

Does CI pull/consume RPMs from here: http://download.eng.bos.redhat.com/rcm-guest/puddles/RHAOS/AtomicOpenShift/3.11/latest/x86_64/os/Packages/haproxy18-1.8.26-1.el7.x86_64.rpm?

If not, it seems that we don't run CI against the same version of haproxy that you get in an installation.

**Comment by frobware** [comment]
/hold

The build output [1] shows that the openshift enterprise repository gets disabled:

```
+ cd /home/origin
+ sudo yum-config-manager --disable 'centos-paas-sig-openshift-origin*-rpms'
Loaded plugins: amazon-id, rhui-lb
+ sudo yum-config-manager --disable 'origin-deps-rhel7*'
Loaded plugins: amazon-id, rhui-lb
+ sudo yum-config-manager --disable 'rhel-7-server-ose*'
Loaded plugins: amazon-id, rhui-lb
```

which means that we won't pick up haproxy18-1.8.26. This slack thread (https://coreos.slack.com/archives/CB95J6R4N/p1603454558311600?thread_ts=1603121486.159700&cid=CB95J6R4N) confirms that the RPM is present in https://mirror.ops.rhcloud.com/enterprise/all/3.11/latest/x86_64/os. This means we are picking up a.n.other version of haproxy18 (i.e., 1.8.1) and it also means we are not testing or consuming the correct version in CI tests.

[1] https://storage.googleapis.com/origin-ci-test/pr-logs/pull/25542/test_pull_request_origin_end_to_end_311/1620/build-log.txt

**Comment by miheer** [comment]
Yes as per our discussion on slack we saw attr  was not  getting supported due to the old version of haproxy getting used failing the CI .

**Comment by frobware** [comment]
/retest

I want to know which haproxy we get now that https://github.com/openshift/origin/pull/24958 just merged.

**Comment by frobware** [comment]
> /retest
> 
> I want to know which haproxy we get now that #24958 just merged.

1.8.23 and that's because we now pick up the version from the net result of merging https://github.com/openshift/origin/pull/24958.

I would say this is also not what we want. If the Centos8 mirror runs and picks up 1.8.26 then this would appear to magically work, until the next time. We need to disable the repo that was added by #24958 for origin CI builds:

```
[centos-openshift-origin]
name=CentOS OpenShift Origin
baseurl=http://mirror.centos.org/centos/7/paas/x86_64/openshift-origin311/
enabled=1
gpgcheck=1
gpgkey=file:///etc/pki/rpm-gpg/RPM-GPG-KEY-CentOS-SIG-PaaS
```

and still work through getting the RPM from the _our_ repo. For Origin/CI we really want a reproducible build and we've just moved away from that.

**Comment by frobware** [comment]
BZ tracking issues related to consuming/pulling haproxy18 in origin CI builds: https://bugzilla.redhat.com/show_bug.cgi?id=1891026

**Comment by frobware** [comment]
```
Extended: [Conformance][templates] templateinstance readiness test should report ready soon after all annotated objects are ready [Suite:openshift/conformance/parallel/minimal] 
```

is failing/flaking a n this PR and others:

https://search.ci.openshift.org/?search=%5C%5BConformance%5C%5D%5C%5Btemplates%5C%5D+templateinstance+readiness+test++should+report+ready+soon+after+all+annotated+objects+are+ready+%5C%5BSuite%3Aopenshift%2Fconformance%2Fparallel%2Fminimal%5C%5D&maxAge=48h&context=1&type=bug%2Bjunit&name=&maxMatches=5&maxBytes=20971520&groupBy=job

**Comment by frobware** [comment]
The `e2e-gcp` is also flaking with:

    Extended: [Conformance][Area:Networking][Feature:Router] The HAProxy router should expose prometheus metrics for a route [Suite:openshift/conformance/parallel/minimal]" 

which is this BZ https://bugzilla.redhat.com/show_bug.cgi?id=1857409

File: images/router/haproxy/Dockerfile [hunk 1]
```diff
@@ -1,29 +1,19 @@
-#
```
Comment (frobware) [inline images/router/haproxy/Dockerfile:1]: Let's resurrect this comment.

**Comment by frobware** [comment]
```
could not wait for RPM repo server to deploy: could not wait for deployment: the deployment config ci-op-ffhlh6q7/rpm-repo failed with status "ProgressDeadlineExceeded"
```

/retest

**Comment by frobware** [comment]
/lgtm

Likely to fail in CI until the fix for https://bugzilla.redhat.com/show_bug.cgi?id=1892775 merges (https://github.com/openshift/origin/pull/25647).

**Comment by frobware** [comment]
This is ready to go now.

/assign @knobunc

**Comment by frobware** [comment]
/hold

Waiting for https://github.com/openshift/origin/pull/25647

It will continue to fail in CI until that merges.

**Comment by frobware** [comment]
Also blocked on https://bugzilla.redhat.com/show_bug.cgi?id=1893657

**Comment by frobware** [comment]
Copying @jkroepke as a matter of courtesy as this PR will revert https://github.com/openshift/origin/pull/24958.

**Comment by frobware** [comment]
/hold cancel

https://github.com/openshift/origin/pull/25647 merged and https://bugzilla.redhat.com/show_bug.cgi?id=1893657 was fixed by #25647.

**Comment by frobware** [comment]
```

Extended: [Area:Networking] services basic functionality should allow connections to another pod on the same node via a service IP [Suite:openshift/conformance/parallel] expand_less | 1m7s
-- | --
/tmp/openshift/build-rpms/rpm/BUILD/origin-3.11.0/_output/local/go/src/github.com/openshift/origin/test/extended/networking/services.go:14 Expected success, but got an error:     <exec.CodeExitError>: {         Err: {             s: "error running &{/usr/bin/kubectl [kubectl --server=https://internal-api.ci-op-n8mzi2lp-7a04a.origin-ci-int-gce.dev.rhcloud.com:8443 --kubeconfig=/tmp/admin.kubeconfig exec --namespace=e2e-tests-net-services1-j66cg execpod-sourceip-ci-op-n8mzi2lp-7a04a-ig-n-95c9ppxtf -- /bin/sh -c wget -T 30 -qO- 172.30.172.195:8080] []  <nil>  wget: can't connect to remote host (172.30.172.195): No route to host\ncommand terminated with exit code 1\n [] <nil> 0xc422997f80 exit status 1 <nil> <nil> true [0xc420cf0a70 0xc420cf0a88 0xc420cf0aa0] [0xc420cf0a70 0xc420cf0a88 0xc420cf0aa0] [0xc420cf0a80 0xc420cf0a98] [0x921b60 0x921b60] 0xc422a5f140 <nil>}:\nCommand stdout:\n\nstderr:\nwget: can't connect to remote host (172.30.172.195): No route to host\ncommand terminated with exit code 1\n\nerror:\nexit status 1\n",
```

/retest

**Pull request merged**